contracts/pm/mixins/MixinReserve.sol

```diff
@@ -42,6 +42,24 @@ contract MixinReserve is MContractRegistry, MReserve {
         info.thawRound = reserve.freezeRound == 0 ? 0 : reserve.freezeRound.add(freezePeriod);
     }
 
+    /**
+     * @dev Returns the amount of funds claimable by a claimant from a reserve
+     * @param _reserveHolder Address of reserve holder
+     * @param _claimant Address of claimant
+     * @return Amount of funds claimable by `_claimant`from the reserve for `_reserveHolder`
+     */
+    function claimableReserve(address _reserveHolder, address _claimant) public view returns (uint256) {
+        ReserveManager storage manager = reserveManagers[_reserveHolder];
+        Reserve storage reserve = manager.reserve;
+        if (reserveState(_reserveHolder) == ReserveState.NotFrozen) {
+            // TODO: Check if claimant is registered in current round
+            return reserve.fundsAdded.div(bondingManager().getTranscoderPoolSize());
+        } else {
+            // TODO: Check if claimant is registered during the freezeRound
+            return reserve.fundsAdded.div(reserve.recipientsInFreezeRound).sub(manager.claimedPerReserve[manager.reserveNonce][_claimant]);
```

Comment:
Same comment as above about returning 0 if `reserve.recipientsInFreezeRound` is 0.